podman_compose.py

```diff
@@ -1922,12 +1972,55 @@ def get_excluded(compose, args):
     if args.services:
         excluded = set(compose.services)
         for service in args.services:
-            excluded -= compose.services[service]["_deps"]
+            excluded -= set(compose.services[service]["_deps"].keys())
             excluded.discard(service)
     log("** excluding: ", excluded)
     return excluded
 
 
+@wait
+def wait_healthy(compose, container_name):
+    info = json.loads(compose.podman.output([], "inspect", [container_name]))[0]
+
+    if not info["Config"].get("Healthcheck"):
+        raise ValueError("Container %s does not define a health check" % container_name)
+
+    health = info["State"]["Healthcheck"]["Status"]
```

Comment:
```suggestion
    health = info["State"]["Health"]["Status"]
```

`Healthcheck` is undefined, at least on `podman > 4.1`.

Reply:
good catch, thanks